configure.ac

```diff
@@ -66,6 +66,7 @@ AC_CHECK_HEADERS([uthash.h])
 AC_CONFIG_FILES([Makefile
 libubx/Makefile
 libubx/ubx0.pc
+libubx/rtl0.pc
```

Comment:
This is a very generic name. To avoid name potential clashes I would suggest to rename to ubxrtl.

Reply:
Done